Examples/Algorithms/Geant4/src/GeantinoRecording.cpp

```diff
@@ -22,6 +22,7 @@
 #include "SteppingAction.hpp"
 
 using namespace ActsExamples;
+using namespace ActsExamples::Geant4;
```

Comment:
Is it safe to use it here?

Reply:
Ah yes. I think this should be safe here. I'm doing this here because `GeantinoRecording` is not inside the `Geant4` namespace. This could also be changed, but I think all other algorithms are directly in `ActsExamples`.